src/app/request-copy/email-request-copy/themed-email-request-copy.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { ThemedComponent } from 'src/app/shared/theme-support/themed.component';
+
+import { EmailRequestCopyComponent } from './email-request-copy.component';
+import { RequestCopyEmail } from './request-copy-email.model';
+
+/**
+ * Themed wrapper for email-request-copy.component
+ */
+@Component({
+  selector: 'ds-themed-email-request-copy',
+  styleUrls: [],
+  templateUrl: './../../shared/theme-support/themed.component.html',
```

Comment:
This path also seems wrong as it starts with a single dot.  I think that first segment can be removed?